connector/src/main/scala/com/microsoft/kusto/spark/utils/CslCommandsGenerator.scala

```diff
@@ -27,6 +27,14 @@ private[kusto] object CslCommandsGenerator {
     s".drop tables ($tables) ifexists"
   }
 
+  def generateShowTableIngestionBatchingPolicyCommand(tableName: String): String = {
+    s".show table ${KustoQueryUtils.normalizeTableName(tableName)} policy ingestionbatching"
+  }
+
+  def generateAlterTableIngestionBatchingPolicyCommand(tableName: String, targetTableBatchingPolicy: String): String = {
+    s""".alter table ${KustoQueryUtils.normalizeTableName(tableName)} policy ingestionbatching @"$targetTableBatchingPolicy""""
```

Comment:
Per discussion with @yogilad:
Nobody cares what the batching policy is of the temp table. Instead, we just want to make sure that the data gets to the destination table without an unexpected (e.g. 5m) delay. So we can set for the temp table a hardcoded very low batching policy (e.g. 5s).
Consider using the file size and number of file batching policies of the destination table (as I already do in this PR) so we don't create unoptimized extents, and only hardcode the time component.

The above assumes we only start this process after we reached the end of their data - confirm this.